flutter/test/sentry_flutter_util.dart

```diff
@@ -6,36 +6,39 @@ import 'package:sentry/sentry.dart';
 import 'package:sentry_flutter/sentry_flutter.dart';
 import 'package:sentry_flutter/src/file_system_transport.dart';
 import 'package:sentry_flutter/src/version.dart';
+
 import 'mocks.dart';
 
-FutureOr<void> configurationTester(
-  SentryOptions options, {
-  bool isWeb = false,
-}) async {
-  options.dsn = fakeDsn;
+FutureOr<void> Function(SentryOptions) getConfigurationTester(bool onIOS) =>
```

Comment:
`bool isWeb = false` is gone, why? the idea is to test if it's running thru a `Web` integration as well, not done yet, but the flag would be necessary I guess

Reply:
as the param wasn't used I removed it for now, should I add it ?

Reply:
✅ I put it back